usecase/usecase-webapi-xwalk-tests/samples/IAPGooglePlay/iap/manifest.json

```diff
@@ -1,12 +1,12 @@
-{
-  "name": "iaptest",
-  "start_url": "index.html",
-  "xwalk_package_id": "org.xwalk.iap",
-  "xwalk_extensions": ["."],
-  "xwalk_android_permissions": [
-    "ACCESS_NETWORK_STATE",
-    "ACCESS_WIFI_STATE",
-    "INTERNET",
-    "IAP"
-  ]
-}
+{
+  "name": "IAP Google Play Demo",
+  "start_url": "iap.html",
+  "xwalk_package_id": "org.crosswalk.iapsample",
```

Comment:
the package id should be com.crosswalk.iapsample. Please read https://github.com/crosswalk-project/crosswalk-android-extensions/tree/master/examples/iap